subsys/fw_info/fw_info.c

```diff
@@ -6,84 +6,150 @@
 
 #include "fw_info.h"
 #include <linker/sections.h>
+#include <sys/util.h>
+#include <init.h>
 #include <errno.h>
 #include <string.h>
 #include <nrfx_nvmc.h>
 
 
-extern const u32_t _image_rom_start;
-extern const u32_t _flash_used;
+extern const u32_t _image_rom_start[];
```

Comment:
Maybe we should add a comment here stating that these are from the linkerscript? I guess not all readers will be known with this scheme.

Reply:
Good idea, added.